bin/matrixMarket2csv.R

```diff
@@ -31,15 +31,15 @@ runs_j<-read.table(file=gzfile(opt$runs_path),
            colClasses = c("NULL","character"))
 # Traverse tpm_mtrx object writing sequentially on an object that we write to disk, appending everynow and then.
 
-genes_per_it<-opt$genes_step
-genes_steps<-seq(1,nrow(genes_i),genes_per_it)
-num_genes<-nrow(genes_i)
-for(g_i in genes_steps) {
-  for(r_j in 1:nrow(runs_j)) {
-      up_to<-min(g_i+genes_per_it-1,num_genes)
-      write.table(data.frame(exp_acc=opt$exp_id,gene_id=genes_i$gene[g_i:up_to],
-                     cell_id=runs_j$run[r_j],expression=tpm_mtrx[g_i:up_to,r_j]),
-                row.names = FALSE, col.names = FALSE, file = opt$output_path, sep = ",",
-                append = TRUE, quote = FALSE)
-  }
-}
\ No newline at end of file
+# Skip first two lines (is this always the format?)
```

Comment:
It is! https://math.nist.gov/MatrixMarket/formats.html

Reply:
Great!